web/libs/ui/src/tailwind.css

```diff
@@ -79,3 +79,9 @@ h6 { font-size: 0.67em; }
     @apply text-primary-content;
   }
 }
+
+@layer utilities {
+  .text-shadow-button {
```

Comment:
sounds like there is a new shadow token that should be used instead of `--black-raw`
cc @ricardoantoniocm @bmartel

Reply:
it's actually there already. i just migrated to it today